evidence/darkfriend77/0001-promote-rank-1-012024.md

```diff
@@ -0,0 +1,124 @@
+# Evidence-0001: Promotion to Rank 1
+
+|                 |                                                                                             |
+| --------------- | ------------------------------------------------------------------------------------------- |
+| **Report Date** | Date of submission (2025/01/09)                                                             |
+| **Submitted by**| Cedric Decoster                                                                             |
+
+
+## Member details
+
+- Matrix username: @roxontox:matrix.org
+- Polkadot address: 14MVAs6jPc7bPW33fC7eTwz4Zf1pD3KWW4tQtrHWUd9w1Phx
+- Current rank: 0
+- Date of initial induction: 2025/01/09
+- Date of last report: N/A
+- Area(s) of Expertise/Interest:
+  - Gaming & Simulations
+  - C# API / SDK
+  - Game Engines (Unity / Unreal)
+  - Automations & Bots
+
+
+## Reporting period
+
+- Start date: 2020/09/07
+- End date: 2025/01/09
+
+
+## Evidence
+*Explain why your contributions in relation to the Polkadot SDK are worthy of retention/promotion. Refer to the terms in Section 6 of the [Manifesto](https://github.com/polkadot-fellows/manifesto/blob/main/manifesto.pdf) and provide links to relevant content (i.e code, articles, media, etc.) to show that you are meeting all the requirements.*
+
+I have been actively contributing to the Substrate ecosystem since 2020 ([fist commit](https://github.com/ajuna-network/SubstrateNetApi/commit/6783d6e9c838045548c7f0d4403e3a019cb7c5a8)), working on open-source repositories and projects such as Ajuna Network, SubstrateGaming, and PolkadotPlay. As Founder and Core Contributor of Ajuna Network, and as core developer behind the entire Substrate C# Stack, including the Polkadot SDK for Unity.
+
+---
+
+### Key Contributions
+Github handle: https://github.com/darkfriend77
+
+**Substrate C# Stack**
+* https://github.com/SubstrateGaming/Substrate.NET.API
+* https://github.com/SubstrateGaming/Substrate.NET.Toolchain
+* https://github.com/SubstrateGaming/Substrate.NET.Wallet
+* https://github.com/SubstrateGaming/Substrate.Chains.NET
+* (Archived) https://github.com/ajuna-network/SubstrateNetApi
+* https://www.nuget.org/profiles/SubstrateGaming
+
+**Polkadot SDK for Unity**
+* https://github.com/PolkadotPlay/Polkadot.Unity.SDK
+
+**Ajuna Network** (https://github.com/ajuna-network):
+* https://github.com/ajuna-network/pallet-ajuna-connectfour
+* https://github.com/ajuna-network/pallet-ajuna-matchmaker
+* https://github.com/ajuna-network/pallet-ajuna-gameregistry
+* https://github.com/ajuna-network/pallet-ajuna-rps
+* https://github.com/ajuna-network/pallet-dotmog
+* https://github.com/ajuna-network/ajuna-pallets
+
+**Web3 Grants**
+* https://github.com/w3f/Grants-Program/pull/265
+* https://github.com/w3f/Grants-Program/pull/433
+* (with PolyCrypt) https://github.com/w3f/Grants-Program/pull/893
```

Comment:
Please either rephrase to provide context and meaning to these links or simply integrate them (or just the big impact ones) in the other well described sections of the letter.

Except this section of links, everything reads well and I can easily understand the context and value added 👍